src/components/Navbar/index.tsx

```diff
@@ -9,30 +9,99 @@ import CommentSummaryButton from '../CommentSummaryButton';
 import LoginButton from '../LoginButton';
 import { ApplicationState } from '../../reducers';
 import { Comment, selectVersionComments } from '../../reducers/comments';
-import { Version, selectCurrentVersionInfo } from '../../reducers/versions';
+import {
+  Version,
+  fetchVersion,
+  getVersionInfo,
+  selectCurrentVersionInfo,
+} from '../../reducers/versions';
 import { ConnectedReduxProps } from '../../configureStore';
 import { User, selectCurrentUser, requestLogOut } from '../../reducers/users';
 import styles from './styles.module.scss';
 
 export type PublicProps = {
+  _nextBaseVersionImprint?: string;
```

Comment:
I gather this was added to support Storybook? I wonder if it would be better to set up the data so that this could just happen naturally with Storybook, rather than adding this "fake" property?

Reply:
I had trouble with this but I figured out a way to do it ✨